packages/components/src/components/chat-search/IntelliJNotice.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div>
+    <button @click.prevent="addFiles">&#xFF0B; add files</button>
```

Comment:
This should be an SVG, otherwise we have no control over how this character is rendered
![image](https://github.com/user-attachments/assets/bef04ac5-1768-44cb-9a10-529d0648f676)

Reply:
Also, we can't have unstyled buttons. At least use the `v-button` component.